### ISSUE TITLE
ML Module: Optimization of network

### ISSUE THREAD

**Issue by keshavshankar08** [keshavshankar08/ArmSense#25]
There is a need to get rid of tensorflow to optimize speed and storage of libraries on the pi for the application

- Generalize and error check for data cleaning and normalization more efficiently
- Develop methods in trainer for neural network training from scratch
- Develop methods in predictor for neural network testing from scratch
- This involves logging software for output of training progress.
